webpack.config.ts

```diff
@@ -4,6 +4,9 @@ const path = require('path')
 module.exports = {
   entry: './src/graphqelm.ts',
   target: 'node',
+  node: {
+    __dirname: false
```

Comment:
Without this, the value of `__dirname` is just `'/'`. It's counter-intuitive that setting it to `false` injects the value. [Reference](https://github.com/webpack/webpack/issues/1599#issuecomment-186841345)